swift_package/BrainFlowCI/BrainFlowBindings/BoardShim.swift

```diff
@@ -0,0 +1,582 @@
+//
+//  BoardShim.swift
+//  a Swift binding for BrainFlow's board_shim high-level API
+//
+//  Created by Scott Miller for Aeris Rising, LLC on 8/23/21.
+//
+
+import Foundation
+
+struct BoardShim {
+    let boardId: BoardIds
+    let masterBoardId: BoardIds
+    let bfParams: BrainFlowInputParams
+    private let jsonBrainFlowInputParams: [CChar]
+    
+    /*
```

Comment:
remove commented out code block

Reply:
I think this one is already resolved. I do not see the commented code block in my local repo.

Reply:
https://github.com/ScottThomasMiller/brainflow/blob/swift-bindings/swift_package/BrainFlowCI/BrainFlowBindings/BoardShim.swift#L16 its here